SQL/New_patches/2020_06_16_Add_Date_Format_to_ConfigSettings_DataType.sql

```diff
@@ -8,4 +8,21 @@ UPDATE ConfigSettings
 
 -- Update DataType of dodFormat
 UPDATE ConfigSettings
-	SET DataType='date_format' WHERE Name='dodFormat';
\ No newline at end of file
+	SET DataType='date_format' WHERE Name='dodFormat';
+
+-- Convert old date casing combos to supported format
+UPDATE Config SET Value='Ymd' 
+	WHERE Value IN ('YMD', 'YMd', 'YmD', 'yMD', 'yMd', 'ymD', 'ymd') 
```

Comment:
This might be a little bit cleaner
```suggestion
	WHERE LOWER(Value)='ymd' 
```

Reply:
@CamilleBeau beast. implemented.

Reply:
@CamilleBeau DAYUM !